### DISCUSSION TITLE
main.goの`log.Printf("start with: %v", url)`はエラー出力でしょうか

### DISCUSSION THREAD

**Discussion by uekiGityuto** [budougumi0617/go_todo_app#69] [Q&A]
執筆ありがとうございます。
Go言語で疑問に感じていた箇所がほぼ全て解消でき、仕組みや理由を理解することができました。
痒いところに手が届くような本で、大変勉強になりました。

一つ質問です。
以下の箇所では、log.PrintfでServerのurlを表示していると思いますが、log.Printfはデフォルトでは標準エラー出力に書き出すものだと思います。このメッセージはinfoレベルのメッセージだと思うのですが、標準エラー出力に書き出すのは何故でしょうか？
https://github.com/budougumi0617/go_todo_app/blob/2384db97e1aea9c02718b089f69af52b2d004e26/main.go#L30

**Comment by budougumi0617** [comment]
ご購入ありがとうございます。参考になった部分が少しでもあれば私としてはとても嬉しいです。

ご質問についてもありがとうございます。
正直に回答すると、"「`リスト16.4　「config」パッケージを利用するように修正した「run」関数`」でプロトタイプコードを追記したときのまま `log` パッケージになっている。"という回答になります。

本来は `logger` を実装して構造化したログを標準出力すべきだと考えます。（そしてサイドカーで用意したfluentdでログを転送するなど）
書籍ではそのあたりまでは触れられていません 🙇

**Comment by uekiGityuto** [comment]
承知しました！
本来はzapなどを使う感じですかね。
ありがとうございます！
